### PR TITLE
fix: use terminateWithManifestConfigError for handling configuration …

### PR DESCRIPTION
**Pull request by rpiszczatowski**
…errors of DEX fetchers

### CODE REVIEW

File: packages/oracle-node/src/fetchers/dex-on-chain/DexOnChainFetcher.ts [hunk 1]
```diff
@@ -5,17 +5,22 @@ import {
...
 export interface Responses<T> {
   [spotAssetId: string]: T;
 }
 
 export abstract class DexOnChainFetcher<T> extends MultiRequestFetcher {
-  abstract calculateLiquidity(assetId: string, response: T): number;
-  calculateSlippage(assetId: string, response: T): number {
-    throw new Error("method not implemented!");
+  calculateLiquidity(_assetId: string, _response: T): number {
+    terminateWithManifestConfigError(`liquidity calculation not implemented for ${this.getName()}`);
```
Comment (hatskier) [inline packages/oracle-node/src/fetchers/dex-on-chain/DexOnChainFetcher.ts:16]: What do you think about renaming the function to smth like `terminateNodeWithError`? IMO currently it's a bit misleading why we use it here, since these errors are not related to the manifest

Comment (koonopek) [reply packages/oracle-node/src/fetchers/dex-on-chain/DexOnChainFetcher.ts:16]: Why do you think the are not related to manifest? In.my opinion they are caused because someone used "toke" in manifest which is not supported by node

Comment (rpiszczatowski) [reply packages/oracle-node/src/fetchers/dex-on-chain/DexOnChainFetcher.ts:16]: You can think of those errors as manifest-related because manifest is the place where you declare the combination of a token name (e.g. OHM_xxx_liquidity) and source that does not support liquidity calculation. This is not a problem with a source that it does not support liquidity calculation.

Comment (cehali) [reply packages/oracle-node/src/fetchers/dex-on-chain/DexOnChainFetcher.ts:16]: I agree with Mike, it is related to a misconfiguration in the manifest. We can change the error message slightly to show there is a problem in manifest like: `liquidity calculation not implemented for ${this.getName()}, you defined liquidity for a token in the manifest with fetcher which doesn't support liquidity`.

**Review by koonopek — APPROVED** [review]
Small step for humanity giant leap for redstone ~ Mike Armstrong
